SupportClient/SupportClient/Views/FeedbackView.swift

```diff
@@ -0,0 +1,295 @@
+        let imagePicker = ImagePickerController()
+        imagePicker.settings.theme.selectionStyle = .checked
+        imagePicker.settings.fetch.assets.supportedMediaTypes = [.image, .video]
+        currentViewController()!.presentImagePicker(imagePicker,
+            select: nil,
+            deselect: nil,
+            cancel: nil,
+            finish: { assets in
+                self.attachedFiles = Array(Set(self.attachedFiles + assets.map { $0 }))
+                self.attachmentsCollectionView.reloadData()
+                self.checkStatus()
+            },
+            completion: nil
+        )
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+}
+
+
+// MARK: - UICollectionView DataSource
+extension FeedbackView: UICollectionViewDataSource {
+    
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return attachedFiles.count
+    }
+        
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: FileCollectionViewCell.reuseID, for: indexPath) as! FileCollectionViewCell
+        let file = attachedFiles[indexPath.item]
+        cell.configure(for: file, with: self)
+        return cell
+    }
+    
+    
+    
+}
+
+// MARK: - UICollectionView Delegate
+extension FeedbackView: UICollectionViewDelegate {
+        
+    // TODO: Open up image editor with image tapped on
+    
+}
+
+// MARK: - UICollectionView DelegateFlowLayout
+extension FeedbackView: UICollectionViewDelegateFlowLayout {
+    
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+        let file = attachedFiles[indexPath.item]
+        let width: CGFloat = CGFloat(Int(imageHeight) * file.pixelWidth / file.pixelHeight)
+        return CGSize(width: width, height: imageHeight)
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, minimumLineSpacingForSectionAt section: Int) -> CGFloat {
+        return padding
+    }
+        
+}
+
+// MARK: - UITextView Delegate
+extension FeedbackView: UITextViewDelegate {
+    
+    func textViewDidBeginEditing(_ textView: UITextView) {
+        // Once user begins editing, change color to text for pseudo-placeholder effect
+        if textView == messageTextView {
+            if textView.textColor == UIColor.lightGray {
+                textView.text = nil
+                textView.textColor = .black
+            }
+        }
+    }
+    
+    func textViewDidEndEditing(_ textView: UITextView) {
+        // If user did not add message, replace with default text
+        if textView == messageTextView {
+            if textView.text.isEmpty {
```

Comment:
lets chain these if statements and also shorten the enum comparison:
`if textView == messageTextView, textView.textColor == .lightGray`